projects/plugins/protect/src/js/components/onboarding-popover/index.jsx

```diff
@@ -29,6 +32,11 @@ const OnboardingPopover = ( { id, anchor, position } ) => {
 		return null;
 	}
 
+	// do not render if the scan is in progress
+	if ( isScanInProgress( status ) ) {
+		return null;
+	}
+
```

Comment:
We should handle the need to hide certain onboarding popovers at the implementation level, i.e. not rendering the specific `<OnboardingPopover>` components in the first place.

These are reusable components, and not every implementation should need to consider the current scan state for example.

Reply:
Good point, IIRC there is only one scenario where there is an `OnboardingPopover` that renders in the `AdminSectionHero` (the one you originally noted causes some awkwardness), but the initial popover applies to both free and paid flows. I am thinking we would only want to hide both in the free flow, but continue to have them display in the paid flow (while scanning)?

Reply:
Yeah, if we can remove this check inside the component itself, and then ensure we aren't rendering the specific onboarding popovers inside the scan results while the scan is in progress, we should be good :+1: